Documentation/install/azure/azure-private.md

```diff
@@ -0,0 +1,56 @@
+# Azure private clusters
+
+Use Tectonic Installer to create clusters within an existing private Azure network. In addition to the standard Azure requirements, private clusters also require:
+* An existing Virtual Network with a pre-configured and connected VPN link.
+* A DNS domain pre-registered into Azure DNS, or a 3rd party DNS server that supports DNS UPDATE operations.
+
+Once these requirements are met, follow the guide to [Install Tectonic on Azure with Terraform][azure-terraform].
+
+## Connected VPN link
+
+Private Azure networks require that an external VPN be available and running prior to running Tectonic Installer.
+
+For more information, see the Microsoft Azure document [Connect an on-premises network to Azure using a VPN gateway][connect-azure].
+
+## DNS domain or server
+
+Private clusters must use an external DNS provider, rather than Azure’s default DNS implementation.
```

Comment:
I'd rather say "default hostname resolution" here. We want to distinguish this from the actual Azure managed DNS service, which CAN be used for private clusters.